### PR TITLE
Convert G* operations to new op form

### PR DESCRIPTION
**Pull request by s-barannikov**
Also includes some quick-fixes for previously converted ops.

### CODE REVIEW

**Comment by aprocter** [comment]
@s-barannikov BTW, our internal CI system is not currently set up to handle pull requests from forks, so we will have to do some manual work to merge it when folks are back in the office on Monday.

**Comment by s-barannikov** [comment]
@aprocter
No problem 😊 Should I create a branch in the upstream next time or continue with the fork?

**Comment by aprocter** [comment]
@s-barannikov Nope, this is fine---this is our standard procedure for external contribs, at least for now.

File: src/ngraph/op/add.cpp [hunk 1]
```diff
@@ -49,7 +49,7 @@ void op::Add::generate_adjoints(autodiff::Adjoints& adjoints, const NodeVector&
     adjoints.add_delta(y, delta);
 }
```
Comment (diyessi) [inline src/ngraph/op/add.cpp]: I ran into problems the last time I tried this change; we might drop the operator+ part if they show up in merge CI.

**Comment by diyessi** [comment]
Migrated to #3278

**Pull request closed without merging**